articles/TOC.yml

```diff
@@ -1,13 +1,14 @@
 - name: Welcome
   href: index.md
-- name: Getting Started
-  topicHref: install-guide/index.md
-  href: install-guide/toc.yml
 - name: Quantum computing concepts
   topicHref: concepts/index.md
   href: concepts/toc.yml
-- name: Quickstart - your first quantum program
-  href: quickstart.md
+- name: Quickstarts
+  items:
+  - name: Install the Quantum Development Kit
```

Comment:
Apologies for the delay in reviewing, but just a few visual sidebar-related comments.

I'm unsure if it's my particular browser config, but this title is just long enough that "Kit" drops to a new line and makes the whole thing look quite clunky (see screengrab below). Perhaps "Install the QDK", but I know we not might want the acronym there. Just plain "Installation" seems rather nondescript. 
While we're at it, "Managing quantum machines and drivers" does the same, cluttering up that important-for-information area of the sidebar (as opposed to "Contributing..." which is more balanced and out of the way down below). Maybe one word less there?
![installQDK](https://user-images.githubusercontent.com/47889725/65715859-362ec480-e09e-11e9-9e15-f06ee4d1718c.JPG)

Reply:
No apologies needed :-) 

I'll experiment with QDK and see what others think.